src/components/index-page/IndexShipFaster.tsx

```diff
@@ -0,0 +1,131 @@
+            report—highlighting the bottlenecks and also offering solutions, starting with the most
+            critical areas first. With these insights, you’ll gain clarity, recapture missed
+            opportunities, and set a definitive course for success.
+          </p>
+        </div>
+        <div className="index-ship-faster__card">
+          <div className="index-ship-faster__tagline">For teams with an existing product</div>
+
+          <div className="index-ship-faster__image-container">
+            <img className="index-ship-faster__image" src="https://placehold.co/104x104" />
+          </div>
+          <h2 className="index-ship-faster__card-heading">Staff Augmentation</h2>
+          <p className="index-ship-faster__text">
+            Too much work and too little time? No one wins when your engineering teams get stretched
+            thin.
+          </p>
+          <p className="index-ship-faster__text">
+            LaunchWare’s Staff Augmentation is not just getting an extra set of hands. It’s about
+            enhancing your team’s capabilities. By seamlessly integrating our skilled professionals
+            into your environment, we don’t just fill the gap – we become an intrinsic part of your
+            team that’s busy getting things shipped.
+          </p>
+          <p className="index-ship-faster__text">
+            We’ll pull from our team of senior and junior developers to make sure we match you with
+            the right people for your project. Then, we’ll immerse ourselves in understanding your
+            codebase, your unique processes, and the milestones you're aiming for. Together, we
+            ensure that your projects not only stay on track but also achieve excellence, every
+            single time.
+          </p>
+        </div>
+        <div className="index-ship-faster__card">
+          <div className="index-ship-faster__tagline">
+            For teams with an idea or concept for a software product
+          </div>
+          <div className="index-ship-faster__image-container">
+            <img className="index-ship-faster__image" src="https://placehold.co/104x104" />
+          </div>
+          <h2 className="index-ship-faster__card-heading">Flight Plan</h2>
+          <p className="index-ship-faster__text">
+            Business leaders brim with innovative software ideas. But it’s not easy to turn a
+            visionary idea into something your team can efficiently execute.
+          </p>
+          <p className="index-ship-faster__text">
+            Enter our Flight Plan engagement—how smart leaders map out their software development
+            journey.
+          </p>
+          {/* <p className="index-ship-faster__text"></p> */}
+          Here’s how it works:
+          <ol className="index-ship-faster__ordered-list">
+            <li>
+              We kick things off with our 'Jobs to Be Done Workshop', honing in on your target
```

Comment:
I think we need to escape the `'` with `&lsquo; &rsquo;`

Reply:
Should I also convert the right quotes that are present throughout the site's copy (e.g. line 82)?